src/components/transport_manager/include/transport_manager/transport_adapter/transport_adapter_impl.h

```diff
@@ -91,7 +91,7 @@ class TransportAdapterImpl : public TransportAdapter,
    * @return true if initialized.
    * @return false if not initialized.
    */
-  virtual bool IsInitialised() const;
+  virtual bool IsInitialised() const OVERRIDE;
```

Comment:
@AByzhynar, @anosach-luxoft, it will be great to spent a half hour and write article with `OVERRIDE` macro and virtual keyword usage =)